src/app/routes/api/v3/admin/forms/admin-forms.submission.routes.ts

```diff
@@ -0,0 +1,43 @@
+import { Router } from 'express'
+
+import * as AdminFormController from '../../../../../modules/form/admin-form/admin-form.controller'
+import * as EncryptSubmissionController from '../../../../../modules/submission/encrypt-submission/encrypt-submission.controller'
+
+export const AdminFormsSubmissionRouter = Router()
+
+/**
+ * Count the number of submissions for a public form
+ * @route GET /:formId/submissions/count
+ * @security session
+ *
+ * @returns 200 with submission counts of given form
+ * @returns 400 when query.startDate or query.endDate is malformed
+ * @returns 401 when user does not exist in session
+ * @returns 403 when user does not have permissions to access form
+ * @returns 404 when form cannot be found
+ * @returns 410 when form is archived
+ * @returns 422 when user in session cannot be retrieved from the database
+ * @returns 500 when database error occurs
+ */
+AdminFormsSubmissionRouter.route(
```

Comment:
can use `get` by itself without the `route` function since we probably have no other HTTP methods on this endpoint

Reply:
i did this on purpose because i felt like the `get` section should solely reflect the middleware functions called and the `route` portion should reflect the http path. 

even though there's only a single handler here, i think this convention would make it easier on the reader because of the semantic meaning attached to `route` and `get`

Reply:
sure, will defer to you.